src/forgot-password/messages.js

```diff
@@ -51,11 +51,6 @@ const messages = defineMessages({
     defaultMessage: 'Enter your email',
     description: 'Error message that appears when user tries to submit empty email field',
   },
-  'forgot.password.invalid.email.message': {
```

Comment:
I see its usage in `ForgotPasswordPage.jsx`. why are we removing it?

Reply:
Can you share the code reference?

Reply:
sorry, it was confusion at my side.